sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -136,10 +136,9 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
              |)""".stripMargin)
 
         spark.sql("REFRESH TABLE t1")
-        // Before SPARK-19678, sizeInBytes should be equal to dataSize.
-        // After SPARK-19678, sizeInBytes should be equal to DEFAULT_SIZE_IN_BYTES.
+        // After SPARK-28573, sizeInBytes should be equal to dataSize.
         val relation1 = spark.table("t1").queryExecution.analyzed.children.head
-        assert(relation1.stats.sizeInBytes === spark.sessionState.conf.defaultSizeInBytes)
+        assert(relation1.stats.sizeInBytes === dataSize)
```

Comment:
cc @wangyum, after changes applied to HiveTableRelation, we don't modify `tableMeta` any more. 
The converted `LogicalRelation(HadoopFsRelation, xx)` from `HiveTableRelation` will actual get size by `HadoopFsRelation.sizeInBytes`. 

Do you think this is an acceptable change or we should deal with stats differently in `DetermineTableStats`

Reply:
looks better to me.